<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportMulFaceInfoDlg</class>
 <widget class="QDialog" name="ImportMulFaceInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批量入库</string>
  </property>
  <widget class="QLabel" name="devIdlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设备ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="devIdlineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ops</string>
   </property>
  </widget>
  <widget class="QComboBox" name="opsComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="audioCheckBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>音频支持</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadFilesBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>加载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>入库</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveFeatBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>保存特征值</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
